site/content/blog/2022-06-17-funding-zap-development/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Funding ZAP Development"
+summary: ""
```

Comment:
I'm totally good with this being something punchy:

> Keep ZAP alive and Simon able to pay bills!

etc

Reply:
"Help Needed: Funding.." ?